queries/fetch_remaining_transactions.sql

```diff
@@ -0,0 +1,19 @@
+SELECT 
+    e.block_number, 
+    e.block_time AS timestamp,
+    e.hash,
+    e."from",
+    e."to",
+    e.value,
+    e.gas_used AS gas,
+    e.gas_price,
```

Comment:
I think for simulation you need `gas_limit` and the EIP-1559 values for the gas price (for Type 2 transactions).